templates/job/detail.html

```diff
@@ -186,17 +188,22 @@ <h1 class="min-w-0 text-3xl break-words md:text-4xl font-bold text-slate-900">
           {% endif %}
         {% /description_item %}
 
-        {% #description_item title="Git commit hash" stacked=True %}
+        {% #description_item title="Git commit hash" %}
           {% link href=job.job_request.get_repo_url text=job.job_request.sha|slice:7|default:"-" %}
         {% /description_item %}
 
-        {% #description_item title="Code comparison" stacked=True %}
+        {% #description_item title="Code comparison" %}
           {% if code_compare_url %}
-            {% link href=code_compare_url text="Compare workspace code to when the last time this action ran on this backend" %}
-            {% if code_compare_succeeded_url %}
-              <br/>
-              {% link href=code_compare_succeeded_url text="Compare workspace code to when the last time this action ran successfully on this backend" %}
-            {% endif %}
+            <ul class="list-disc ml-4 space-y-1">
+              <li>
+                {% link href=code_compare_url text="Compare workspace code to the previous action run on "|add:backendName %}
```

Comment:
I'm not 100% on this now I think about it. The "previous action" might not have been this action, and the comparison URL is specifically for the commits for this run of this action vs the last run of **this** action

Reply:
> Compare workspace code to the previous run of this action on [...]

Or is that even more confusing? "this action" we could text replace the action name?

Reply:
Oh I think you're onto something with putting the action name in there. That feels pretty unambiguous to me.

Reply:
@sebbacon  thoughts on  "Compare workspace code to the previous _my_nifty_action_ run on TPP" or similar?

Reply:
![image](https://github.com/opensafely-core/job-server/assets/25795240/ef6b581a-4737-4af4-84ae-c9cdadb4a7a4)


too much text 👎  It's slightly better if I manually fettle the html to make the action name a monospaced code block but can't seem to get that to place nice with the custom `link` component. Even then it's not great. Some action names can be very long

Reply:
![image](https://github.com/opensafely-core/job-server/assets/25795240/ab46175f-bddf-4768-abc2-120fc59a02c2)

I think I prefer this simple change